src/lib/components/NavBar.svelte

```diff
@@ -1,8 +1,8 @@
 <script lang="ts">
-  import ioxio from "$lib/images/ioxio-logo.svg?dataurl"
-  import github from "$lib/images/github-icon.svg?dataurl"
+  import IoxioLogo from "$lib/images/ioxio-logo.svg?dataurl"
+  import GitHubLogo from "$lib/images/github-icon.svg?dataurl"
   import Container from "$lib/components/Container.svelte"
-  import Hamburger from "$lib/images/hamburger-icon.svg"
+  import HamburgerIcon from "$lib/images/hamburger-icon.svg"
 
   export let popover: any
 </script>
```

Comment:
Can we add a breakpoint observer for `>sm` to `popover.close()` so the popover can't be left to an open state if you change screen size to be larger after opening it?